it/mock-server/src/main/java/com/adobe/cq/testing/mockserver/MockRequest.java

```diff
@@ -0,0 +1,592 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+package com.adobe.cq.testing.mockserver;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.eclipse.jetty.http.HttpMethod;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.Principal;
+import java.util.*;
+
+/**
+ * MockRequest contains the data that is used to match a {@link RequestResponseRule} against an incoming HTTP request.
+ *
+ * @author Mark J. Becker {@literal <mabecker@adobe.com>}
```

Comment:
There are a few of those `@author` JSDOC: I don't mind if you want to leave them, but make sure they only appear at the class level then. Some are set on class methods. Please also check other classes.

Reply:
I will remove them completely.